sky/adaptors/common.py

```diff
@@ -2,10 +2,11 @@
 import functools
 import importlib
 import threading
+from types import ModuleType
```

Comment:
import module instead of class

Reply:
typing is the exception: https://google.github.io/styleguide/pyguide.html#2241-exemptions